ports/espressif/boards/adafruit_feather_esp32c6_4mbflash_nopsram/pins.c

```diff
@@ -66,18 +68,15 @@ static const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_IO7), MP_ROM_PTR(&pin_GPIO7) },
     { MP_ROM_QSTR(MP_QSTR_D9), MP_ROM_PTR(&pin_GPIO7) },
```

Comment:
ugh i dont like Dx names that dont match IOx names, because arduino doesnt do remapping, it will confuse people. just stick to IOx when the 'D' name is not the same please

Reply:
ditto for D13, D12, D11, D10

Reply:
D names are there so code is portable. When printing out the pin object, the top name will be used. Only `dir(board)` will show the D names too.

Reply:
which code are we trying to make portable? Is it Feather code that assumes the SAMD/etc numbering?

Reply:
Yes, any code that assumes the standard feather pinout. https://learn.adafruit.com/adafruit-feather/feather-specification#pin-naming-3096825